docs/root/intro/version_history.rst

```diff
@@ -18,6 +18,8 @@ Version history
 * tracing: the sampling decision is now delegated to the tracers, allowing the tracer to decide when and if
   to use it. For example, if the :ref:`x-b3-sampled <config_http_conn_man_headers_x-b3-sampled>` header
   is supplied with the client request, its value will override any sampling decision made by the Envoy proxy.
+* cli: added --config-yaml flag to the Envoy binary. When set its value is interpreted as a yaml
```

Comment:
nit: please alpha order "cli" in the above list.

Reply:
Missed this comment on the first pass. Done.